Source/Core/Core/IOS/Network/SSL.cpp

```diff
@@ -385,6 +385,7 @@ std::optional<IPCReply> NetSSLDevice::IOCtlV(const IOCtlVRequest& request)
     {
       WII_SSL* ssl = &_SSL[sslID];
       const std::string cert_base_path = File::GetUserPath(D_SESSION_WIIROOT_IDX);
+      cert_base_path.pop_back();
       const std::vector<u8> client_cert =
           ReadCertFile(cert_base_path + "/clientca.pem", s_client_cert_hash, m_cert_error_shown);
```

Comment:
Have you considered removing it from the string instead?
```suggestion
      const std::vector<u8> client_cert =
          ReadCertFile(cert_base_path + "clientca.pem", s_client_cert_hash, m_cert_error_shown);
```

Same for the one below.

(And actually, this should probably use the C++17/20/2z/whatever-equivalent of C#'s `Path.Combine` if I could make a wish; but I don't know if there is anything in `std::filesystem` that does this cheaply)